src/pages/settings/SettingFormCheckbox.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC, useState } from "react";
+import { Input, Button, Icon } from "@canonical/react-components";
+import { LxdConfigOption } from "types/config";
+import { getConfigId } from "./SettingForm";
+
+interface Props {
+  initialValue: boolean;
+  configField: LxdConfigOption;
+  onSubmit: (newValue: string | boolean) => void;
+  onCancel: () => void;
+}
+
+const SettingFormCheckbox: FC<Props> = ({
+  initialValue,
+  configField,
+  onSubmit,
+  onCancel,
+}) => {
+  const [checked, setChecked] = useState(initialValue);
+
+  const canBeReset = String(configField.default) !== String(checked);
+
+  const resetToDefault = () => {
+    setChecked(String(configField.default) === "true");
+  };
+
+  return (
+    <>
+      <Input
```

Comment:
This (and the other inputs) don't have a label. I think for a11y we should always have a label. We could
* use the other table column as a label -- this might be complicated, because the edit mode enabled state is not available in the upper scope of the column. We could always mark the 2nd column as a label. In read mode this would cause orphaned labels, if that is not too bad to keep it simple.
* or introduce a new label locally in the components, But the label is not visible, just for screen readers.

Please check with the a11y experts (maybe Accessibility Guild channel?) what is the better option.